src/pages/About.tsx

```diff
@@ -1,81 +1,34 @@
 import Button from '@material-ui/core/Button';
+import CenterWrapper from 'components/CenterWrapper';
 import Grid from '@material-ui/core/Grid';
 import MailIcon from '@material-ui/icons/Mail';
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
 
-const CenterWrapper: React.FC = ({ children }) => (
-  <Grid container justify="center">
-    <Grid justify="center" xs={10}>
-      {children}
-    </Grid>
-  </Grid>
-);
-
-interface ProfileProps {
-  name: string;
-  surname: string;
-  bio: string;
-  role: string;
-  color: string;
-}
-
-const Profile = ({ name, surname, bio, role, color }: ProfileProps) => (
-  <Grid item md={4} sm={6} xs={12}>
```

Comment:
I liked the dynamic change in width of Profiles better. It better balanced things out.

On a large screen, there is currently one row with 4 items and one with 1 item. It feels more balanced with a first row with 3 items and another row with 2 items.

Reply:
I disagree. Before I felt the images were too large, it looked strange. Now the grid is the same size as the images by default so no resizing is necessary with these new grids

Reply:
I'm glad that you state that you disagree! I am however still against that the images have a set width specified in pixels. This makes the page look good on small devices or devices with a low pixel-ratio, but makes it look bad on larger screens. Just because it looks good on the screen that is being developed on does not mean it looks good on other screens!

Perhaps we can make them relatively smaller compared to the utilized space, (i.e. instead of taking up like 20% they take up 5%)?

Reply:
We can possibly set a max-width on the images that represents the width of the original image. I believe they've been scaled down in this version?